### PR TITLE
support sepolia

### PR DESCRIPTION
**Pull request by mendesfabio**
# Description

Deploys Subgraph to Sepolia

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas

### Merges to `dev`
- [ ] I have checked that the graft base is not a pruned deployment

### CODE REVIEW

File: src/mappings/helpers/constants.ts [hunk 1]
```diff
@@ -72,6 +74,8 @@ function forNetwork(addressByNetwork: AddressByNetwork, network: string): Addres
     return Address.fromString(addressByNetwork.optimism);
   } else if (network == 'avalanche') {
     return Address.fromString(addressByNetwork.avalanche);
+  } else if (network == 'sepolia') {
+    return Address.fromString(addressByNetwork.bnb);
```
Comment (markusbkoch) [inline src/mappings/helpers/constants.ts:78]:
```suggestion
    return Address.fromString(addressByNetwork.sepolia);
```

**Comment by markusbkoch** [comment]
@mendesfabio could we take this opportunity to refactor `constants.ts` so that we don't have to update it for every new network if the vault address is the standard vanity address? This would avoid the redeploys to all the other networks.

**Comment by mendesfabio** [comment]
> @mendesfabio could we take this opportunity to refactor `constants.ts` so that we don't have to update it for every new network if the vault address is the standard vanity address? This would avoid the redeploys to all the other networks.

yea, I agree we could do this, let me put something together.

**Comment by markusbkoch** [comment]
pending assets addresses

**Comment by mendesfabio** [comment]
> pending assets addresses

@markusbkoch just updated with BAL and WETH from [deployment task](https://github.com/balancer/balancer-v2-monorepo/blob/master/pkg/deployments/tasks/00000000-tokens/output/sepolia.json) and stables that @mikemcdonald deployed.

**Pull request merged**